app/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:supportsRtl="true"
+        android:supportsRtl="false"
```

Comment:
Why?

Reply:
because we want the constraints to be from left to right (because our application is on English)